### PR TITLE
Add Greek as supported language

### PR DESCRIPTION
**Pull request by Pana-g**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,9 +15,19 @@ Credits go to [basmilius](https://github.com/basmilius) for the awesome [weather
 
 ## FAQ
 
-- [Why don't I see the current day in my weather forecast?](#why-dont-i-see-the-current-day-in-my-weather-forecast)
-- [Why does the forecast show less days than expected?](#why-does-the-forecast-show-less-days-than-expected)
-- [What does the card actually display?](#what-does-the-card-actually-display)
+- [Clock Weather Card](#clock-weather-card)
```
Comment (pkissling) [inline README.md:18]: was this change intentional? i don't want the full TOC here. only the TOC of FAQ

**Review by pkissling — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Pana-g) [reply README.md]: It was accidentally auto generated by a VS Code extension.

**Review by pkissling — APPROVED** [review]
